app/keepers/keepers.go

```diff
@@ -0,0 +1,481 @@
+	ibctransferkeeper "github.com/cosmos/ibc-go/v4/modules/apps/transfer/keeper"
+	ibctransfertypes "github.com/cosmos/ibc-go/v4/modules/apps/transfer/types"
+	ibccoreclient "github.com/cosmos/ibc-go/v4/modules/core/02-client"
+	ibcclienttypes "github.com/cosmos/ibc-go/v4/modules/core/02-client/types"
+	ibctypes "github.com/cosmos/ibc-go/v4/modules/core/05-port/types"
+	ibchost "github.com/cosmos/ibc-go/v4/modules/core/24-host"
+	ibckeeper "github.com/cosmos/ibc-go/v4/modules/core/keeper"
+	epochskeeper "github.com/persistenceOne/persistence-sdk/v2/x/epochs/keeper"
+	epochstypes "github.com/persistenceOne/persistence-sdk/v2/x/epochs/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/halving"
+	"github.com/persistenceOne/persistence-sdk/v2/x/ibchooker"
+	ibchookerkeeper "github.com/persistenceOne/persistence-sdk/v2/x/ibchooker/keeper"
+	ibchookertypes "github.com/persistenceOne/persistence-sdk/v2/x/ibchooker/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/interchainquery"
+	interchainquerykeeper "github.com/persistenceOne/persistence-sdk/v2/x/interchainquery/keeper"
+	interchainquerytypes "github.com/persistenceOne/persistence-sdk/v2/x/interchainquery/types"
+	"github.com/persistenceOne/pstake-native/v2/x/lscosmos"
+	lscosmoskeeper "github.com/persistenceOne/pstake-native/v2/x/lscosmos/keeper"
+	lscosmostypes "github.com/persistenceOne/pstake-native/v2/x/lscosmos/types"
+	tmos "github.com/tendermint/tendermint/libs/os"
+
+	// unnamed import of statik for swagger UI support
+	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"
+)
+
+type AppKeepers struct {
+	keys    map[string]*storetypes.KVStoreKey
+	tkeys   map[string]*storetypes.TransientStoreKey
+	memKeys map[string]*storetypes.MemoryStoreKey
+
+	AccountKeeper         *authkeeper.AccountKeeper
+	BankKeeper            *bankkeeper.BaseKeeper
+	CapabilityKeeper      *capabilitykeeper.Keeper
+	StakingKeeper         *stakingkeeper.Keeper
+	SlashingKeeper        *slashingkeeper.Keeper
+	MintKeeper            *mintkeeper.Keeper
+	DistributionKeeper    *distributionkeeper.Keeper
+	GovKeeper             *govkeeper.Keeper
+	UpgradeKeeper         *upgradekeeper.Keeper
+	CrisisKeeper          *crisiskeeper.Keeper
+	ParamsKeeper          *paramskeeper.Keeper
+	IBCKeeper             *ibckeeper.Keeper
+	ICAHostKeeper         *icahostkeeper.Keeper
+	EvidenceKeeper        *evidencekeeper.Keeper
+	TransferKeeper        *ibctransferkeeper.Keeper
+	FeegrantKeeper        *feegrantkeeper.Keeper
+	AuthzKeeper           *authzkeeper.Keeper
+	HalvingKeeper         *halving.Keeper
+	WasmKeeper            *wasm.Keeper
+	EpochsKeeper          *epochskeeper.Keeper
+	ICAControllerKeeper   *icacontrollerkeeper.Keeper
+	LSCosmosKeeper        *lscosmoskeeper.Keeper
+	InterchainQueryKeeper *interchainquerykeeper.Keeper
+	TransferHooksKeeper   *ibchookerkeeper.Keeper
+
+	// Modules
+	TransferModule             transfer.AppModule
+	TransferIBCModule          ibctypes.IBCModule
```

Comment:
is this required?

Reply:
this is for cases it is used outside the keeper function. There are usecases of some of these modules, hence tried to keep them